.github/workflows/orca.yml

```diff
@@ -0,0 +1,90 @@
+name: ORCA CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.allow-failure }}
+    env:
+      ORCA_SUT_NAME: drupal/example
+      ORCA_SUT_BRANCH: main
+      ORCA_PACKAGES_CONFIG: example/tests/packages.yml
+      ORCA_PACKAGES_CONFIG_ALTER: example/tests/packages_alter.yml
+      ORCA_ENABLE_NIGHTWATCH: FALSE
+      # Hardcode path since GITHUB_WORKSPACE can't be used here.
+      # @see https://github.community/t/how-to-use-env-context/16975/9
+      ORCA_SUT_DIR: /home/runner/work/orca/example
+      ORCA_SELF_TEST_COVERAGE_CLOVER: $HOME/build/logs/clover-self.xml
+      ORCA_JOB: ${{ matrix.orca-job }}
+
+    strategy:
+      matrix:
+        orca-job:
+          - STATIC_CODE_ANALYSIS
+          - INTEGRATED_TEST_ON_OLDEST_SUPPORTED
+          - INTEGRATED_TEST_ON_LATEST_LTS
+          - INTEGRATED_TEST_ON_PREVIOUS_MINOR
+          - INTEGRATED_UPGRADE_TEST_FROM_PREVIOUS_MINOR
+          - ISOLATED_TEST_ON_CURRENT
+          - INTEGRATED_TEST_ON_CURRENT
+          - INTEGRATED_UPGRADE_TEST_TO_NEXT_MINOR
+          - ISOLATED_TEST_ON_CURRENT_DEV
+          - INTEGRATED_TEST_ON_CURRENT_DEV
+          - STRICT_DEPRECATED_CODE_SCAN
+          - ISOLATED_TEST_ON_NEXT_MINOR
+          - INTEGRATED_TEST_ON_NEXT_MINOR
+          - ISOLATED_UPGRADE_TEST_TO_NEXT_MAJOR_BETA_OR_LATER
+        php-version: ["7.3"]
+        allow-failure: [false]
+        include:
+          - orca-job: ISOLATED_TEST_ON_CURRENT
+            php-version: "8.0"
+            allow-failure: false
+          - orca-job: INTEGRATED_UPGRADE_TEST_TO_NEXT_MINOR_DEV
+            php-version: "7.3"
+            allow-failure: true
+          - orca-job: LOOSE_DEPRECATED_CODE_SCAN
+            php-version: "7.3"
+            allow-failure: true
+          - orca-job: DEPRECATED_CODE_SCAN_W_CONTRIB
+            php-version: "7.3"
+            allow-failure: true
+          - orca-job: ISOLATED_TEST_ON_NEXT_MINOR_DEV
+            php-version: "7.3"
+            allow-failure: true
+          - orca-job: INTEGRATED_TEST_ON_NEXT_MINOR_DEV
+            php-version: "7.3"
+            allow-failure: true
+          - orca-job: ISOLATED_UPGRADE_TEST_TO_NEXT_MAJOR_DEV
+            php-version: "7.3"
+            allow-failure: true
```

Comment:
There are some differences from the corresponding Travis CI jobs here. Hopefully I've got it all straight:

```yaml
strategy:
  matrix:
    orca-job:

      - STATIC_CODE_ANALYSIS # ORCA_COVERALLS_ENABLE=TRUE on Travis CI
      - ISOLATED_TEST_ON_CURRENT # ORCA_ENABLE_NIGHTWATCH=TRUE on Travis CI
      - ISOLATED_TEST_ON_CURRENT_DEV # ORCA_ENABLE_NIGHTWATCH=TRUE on Travis CI
      - ISOLATED_TEST_ON_NEXT_MINOR # ORCA_ENABLE_NIGHTWATCH=TRUE on Travis CI

    include:

      - orca-job: ISOLATED_TEST_ON_CURRENT # ORCA_ENABLE_NIGHTWATCH=TRUE on Travis CI
        php-version: "8.0"
        allow-failure: false

      - orca-job: INTEGRATED_UPGRADE_TEST_TO_NEXT_MINOR_DEV
        php-version: "7.3"
        allow-failure: true # Not allowed to fail on Travis CI

      - orca-job: LOOSE_DEPRECATED_CODE_SCAN
        php-version: "7.3"
        allow-failure: true # Not allowed to fail on Travis CI

      - orca-job: ISOLATED_UPGRADE_TEST_TO_NEXT_MAJOR_DEV
        php-version: "7.3"
        allow-failure: true # Not allowed to fail on Travis CI
```

As to the Coveralls bit... why don't we ignore that for this PR and make a follow-up to move it from Travis CI to Actions once we've got everything else working and live?

Reply:
The builds allowed to fail match what's currently in ORCA's template .travis.yml. What am I missing? e.g. for INTEGRATED_UPGRADE_TEST_TO_NEXT_MINOR_DEV: https://github.com/acquia/orca/blob/9cf8d6ed506d26cf195fdbaa21f7513701fc37a6/example/.travis.yml#L158

Coveralls follow-up support in ORCA-219